packages/ERTP/src/payment.js

```diff
@@ -8,12 +8,10 @@ import { defineKind } from '@agoric/vat-data';
  * @param {Brand<K>} brand
  * @returns {() => Payment<K>}
  */
-export const makePaymentMaker = (allegedName, brand) => {
+export const definePaymentKind = (allegedName, brand) => {
```

Comment:
@FUDCo why is it safe for the kind's behavior to reference this parameter of its enclosing scope? To make this kind durable would `brand` have to go on `state`?

Reply:
Yes, for durability, `brand` would have to go on `state`. Or, at the very least, there'd need to be some persistent way to recover `brand` from the payment.  Depending on the context, a reasonable alternative might be durable map store that mapped from kind handle to the brand, which save you from lots of redundant stores of `brand` at the expense of a little bit of extra complication to extract it.

This does raise an interesting API design question, though, which is whether we should provide a way to go from a durable virtual object to its kind handle.  I can imagine that being useful but I could also imagine there might be ocap issues.

Reply:
It also strikes me as possible that if the invoking context that sets up the kind for a given payment type always passes in the same brand for that type with each incarnation of the vat, then the above code would actually be OK as is for durable kinds.  The key is that each incarnation of the vat already needs code to reassociate each of its existing kinds with their behavior when the vat is restarted, and that code might very well be in a position to perform the kind handle to brand mapping without any special API magic